src/Microsoft.OData.Edm/Csdl/Semantics/CsdlSemanticsModel.cs

```diff
@@ -522,15 +523,40 @@ private IEdmVocabularyAnnotation WrapVocabularyAnnotation(CsdlAnnotation annotat
                 ann => new CsdlSemanticsVocabularyAnnotation(schema, targetContext, annotationsContext, ann, qualifier));
         }
 
-        private void AddSchema(CsdlSchema schema)
+        private void AddSchema(CsdlSchema schema, bool addSchemaElements = true)
         {
-            this.AddSchema(schema, true);
+            this.AddSchema(schema, addSchemaElements, true, null);

```

Comment:
```suggestion
            this.AddSchema(schema, addSchemaElements, true /*addAnnotations*/, null/*includeAnnotations*/);
```

Reply:
Maybe I could use named parameters instead, like:
```c#
this.AddSchema(schema, addSchemaElements, addAnnotations: true, includeAnnotations: null);
```

Reply:
@habbes Use John's suggestion. Named parameters aren't supported in some c# versions.